vpr/src/route/rr_graph.cpp

```diff
@@ -2021,6 +2038,9 @@ static std::function<void(t_chan_width*)> alloc_and_load_rr_graph(RRGraphBuilder
     /* If Fc gets clipped, this will be flagged to true */
     *Fc_clipped = false;
 
+    /* This function is called to build the general routing graph resoruces. Thus, the edges are not remapped yet.*/
+    bool is_remapped = false;
```

Comment:
is_remapped -> switches_remapped.
Change comment to explain that the rr_graph will use switch indices from the architecture file at this point (not remapped).
Or change the name to switches_are_arch and negate all the boolean conditions (and still comment it!).